### PR TITLE
Compute Budget inside Neon-CLI

### PR DESCRIPTION
**Pull request by ai-trepalin**
**Problem**
Neon CLI does not use compute budget statements for messages.

**Proposed Solution**
We should to add compute budget instructions into Neon CLI messages.
After that we must fix python tests to use `TransactionWithComputeBudget` instead of `Transaction`.

### CODE REVIEW

File: evm_loader/tests/test_transaction.py [hunk 1]
```diff
@@ -449,5 +450,84 @@ def test_07_combined_continue_gets_before_the_creation_of_accounts(self):
                                                                           step=Step.Complete)
 
 
+# def test_fail_on_no_signature(self):
```
Comment (s-medvedev) [inline evm_loader/tests/test_transaction.py:453]: Do we need this commented test?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+version: "3"
```
Comment (s-medvedev) [inline docker-compose.yml:1]: We already have `evm_loader/docker-compose-test.yml` file. Why we need this one?

File: evm_loader/tests/test_acc_storage_states.py [hunk 3]
```diff
@@ -125,27 +125,29 @@ def neon_emv_instr_cancel_21(self, acc, caller, storage, nonce):
 
     def call_begin(self, storage, steps, msg, instruction, writable_code, acc, caller, add_meta=[]):
         print("Begin")
-        trx = Transaction()
-        trx.add(self.sol_instr_keccak(make_keccak_instruction_data(1, len(msg), 13)))
+        trx = TransactionWithComputeBudget()
+        self.index = len(trx.instructions)
```
Comment (s-medvedev) [inline evm_loader/tests/test_acc_storage_states.py:129]: IMHO: it is better to rename `self.index` in something more clear. For example `self.first_instruction_index`.

File: evm_loader/tests/test_solidity_precompiles.py [hunk 4]
```diff
@@ -91,6 +93,7 @@ def extract_measurements_from_receipt(self, receipt):
 
         for instr in instructions:
             if instr['program'] in ('KeccakSecp256k11111111111111111111111111111',): continue
+            if instr['program'] in ('ComputeBudget111111111111111111111111111111',): continue
```
[the remark this answers is not in the document]
Comment (ai-trepalin) [reply evm_loader/tests/test_solidity_precompiles.py:96]: Merged with list above

File: docker-compose.yml (hunk 2 above)
Comment (ai-trepalin) [reply docker-compose.yml:1]: Removed

File: evm_loader/tests/test_transaction.py (hunk 1 above)
Comment (ai-trepalin) [reply evm_loader/tests/test_transaction.py:453]: No, in develop branch they already removed

File: evm_loader/tests/test_acc_storage_states.py (hunk 3 above)
Comment (ai-trepalin) [reply evm_loader/tests/test_acc_storage_states.py]: Renamed to `first_instruction_index` here and in other tests too.

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**